### PR TITLE
REUSE compliance and no spaces in file-names #5

### PR DESCRIPTION
**Pull request by hoijui**

### CODE REVIEW

**Review by KarlK90 — APPROVED** [review]

File: .reuse/dep5 [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Files: Bin/*.bin
+Copyright: qmk.fm
+License: GPL-2.0-or-later
+
+Files: PCB/gerbers* docs/yaemk_rev_1_2.html docs/yaemk_*.pdf
+Copyright: Stefan Kerkmann <karlk90@pm.me>
+License: CC-BY-SA-4.0
+
+Files: docs/images/* Case/* PCB/assembly/*
```
Comment (KarlK90) [inline .reuse/dep5:18]: `PCB/assembly/*` is also listed for GPL-3.0-only in line 6, is this intentional?

**Review by KarlK90 — CHANGES_REQUESTED** [review]

**Comment by KarlK90** [comment]
@hoijui Thank you for your contribution, changes look good just some minor things for clarification.

File: .reuse/dep5 [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Copyright: Stefan Kerkmann <karlk90@pm.me>
+License: CC-BY-SA-4.0
+
+Files: PCB/prettylib/*.kicad_mod
+Copyright: Digi-Key Corporation
```
[the remark this answers is not in the document]
Comment (hoijui) [reply .reuse/dep5:23]: AHH!!
thanks!
I just did a web search for . I think one of the files, and the closest I could find was what I linked to 2 lines below:
https://github.com/Digi-Key/digikey-kicad-library/blob/master/LICENSE.md

but ... please feel free to fix it yourself, or tell me which files you did, and which are from the kicad library.

File: .reuse/dep5 (hunk 1 above)
Comment (hoijui) [reply .reuse/dep5:18]: ahh, good spotting!
will remove it from the GPL-3.0-only section

something else: what is the `PCB/assembly/overrides.json` file.. it is also form you, right?

Comment (KarlK90) [reply .reuse/dep5]: This file is for correcting the parts location files that is generated by KiCAD for assembly by JLCPCB. Mostly rotating the ICs that have a different orientation when uploaded to JLC.

File: .reuse/dep5 (hunk 2 above)
[the remark this answers is not in the document]
Comment (KarlK90) [reply .reuse/dep5:23]: @hoijui I hope this is the correct format, to my best effort this is what I came up with. :-)

```
Files:
PCB/prettylib/Crystal_SMD_3225-4Pin_3.2x2.5mm.kicad_mod
PCB/prettylib/D_SOD-123.kicad_mod
PCB/prettylib/D_SOD-523.kicad_mod
PCB/prettylib/Fuse_1206_3216Metric.kicad_mod
PCB/prettylib/LED-0603.kicad_mod
PCB/prettylib/LQFP-48_7x7mm_P0.5mm.kicad_mod
PCB/prettylib/MountingHole_2.2mm_M2_Pad_Via.kicad_mod
PCB/prettylib/PinHeader_1x06_P2.54mm_Horizontal.kicad_mod
PCB/prettylib/RotaryEncoder_Alps_EC11E-Switch_Vertical_H20mm.kicad_mod
PCB/prettylib/SMD-0603.kicad_mod
PCB/prettylib/SMD-0805.kicad_mod
PCB/prettylib/SOLDER_JUMPER_2.kicad_mod
PCB/prettylib/SOLDER_JUMPER_3.kicad_mod
PCB/prettylib/SOT-23-5.kicad_mod
PCB/prettylib/SOT-23-6.kicad_mod
PCB/prettylib/SOT-23.kicad_mod
PCB/prettylib/TSSOP-8_4.4x3mm_P0.65mm.kicad_mod
Copyright: KiCAD
License: CC-BY-SA-4.0

Files:
PCB/prettylib/JLC_SMT_Assembly_Hole.kicad_mod
PCB/prettylib/LED-0603_flipable.kicad_mod
PCB/prettylib/LOGO.kicad_mod
PCB/prettylib/MX_KAILH_SOCKET_1U.kicad_mod
PCB/prettylib/MX_KAILH_SOCKET_2U_2X1U.kicad_mod
PCB/prettylib/SK6812_mini_e_flipable.kicad_mod
PCB/prettylib/SK6812_mini_flipable.kicad_mod
PCB/prettylib/SMD_Tacticle_Switch_TS24CA_flipable.kicad_mod
PCB/prettylib/SMD_Tacticle_Switch_TS24CA.kicad_mod
PCB/prettylib/SSD1107_DISPLAY.kicad_mod
PCB/prettylib/USB_C_Receptacle_MUP_U22401.kicad_mod
Copyright: Stefan Kerkmann <karlk90@pm.me>
License: CC-BY-SA-4.0
```

**Comment by hoijui** [comment]
Thank you stefan for that list of files you did!
I finally got around to putting it into a commit. I think the PR is now good.

**Comment by KarlK90** [comment]
@hoijui Thanks for all the effort, looks good and merged.

**Pull request merged**